src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -1465,6 +1468,21 @@ def _save_params(self):
             extension_group = self._get_zarr_extension_group(mode="r+")
             extension_group.attrs["params"] = check_json(params_to_save)
 
+    def _save_info(self):
```

Comment:
Same comment as for `get_class_info`, this is too vague. Aligned with the above I suggest `save_importing_provenance`.

Reply:
yes